src/pages/book/components/complete/Complete.tsx

```diff
@@ -0,0 +1,49 @@
+import { NAVIGATION_STATE } from "@constants/navigationState";
+import { useHeader } from "@hooks/useHeader";
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import FreeBook from "../freeBook/FreeBook";
+import PaidBook from "../paidBook/PaidBook";
+
+const Complete = () => {
+  const navigation = useNavigate();
```

Comment:
p5 ) 통일성을 위해서

```suggestion
  const navigate = useNavigate();
```

로 바꿔주는 게 좋을 것 같아요,,,,,,,,,, 근데 우리끼리 다 다를 거 같아서 확인 필요할 듯 안 급하니까 나중에 해두 되긴 함 사실 ㅋㅋ